src/plugins/orangehrmBuzzPlugin/Controller/BuzzAnniversaryDataMockController.php

```diff
@@ -0,0 +1,135 @@
+                        'joinedDate' => '2016-10-17',
+                    ],
+                    [
+                        'employee' => [
+                            'empNumber'=> 1,
+                            'lastName' => 'user04',
+                            'firstName'=> 'test',
+                            'deleted'=> false,
+                        ],
+                        'jobTitle' => 'Senior Project Lead',
+                        'joinedDate' => '2020-10-15',
+                    ],
+                    [
+                        'employee' => [
+                            'empNumber'=> 7,
+                            'lastName' => 'user05',
+                            'firstName'=> 'test',
+                            'deleted'=> false,
+                        ],
+                        'jobTitle' => 'Senior Project Lead',
+                        'joinedDate' => '2020-10-15',
+                    ],
+                    [
+                        'employee' => [
+                            'empNumber'=> 9,
+                            'lastName' => 'user06',
+                            'firstName'=> 'test',
+                            'deleted'=> false,
+                        ],
+                        'jobTitle' => 'Senior Project Lead',
+                        'joinedDate' => '2020-10-15',
+                    ],
+                    [
+                        'employee' => [
+                            'empNumber'=> 19,
+                            'lastName' => 'user07',
+                            'firstName'=> 'test',
+                            'deleted'=> false,
+                        ],
+                        'jobTitle' => 'Senior Project Lead',
+                        'joinedDate' => '2020-10-15',
+                    ],
+                    [
+                        'employee' => [
+                            'empNumber'=> 11,
+                            'lastName' => 'user08',
+                            'firstName'=> 'test',
+                            'deleted'=> false,
+                        ],
+                        'jobTitle' => 'Senior Project Lead',
+                        'joinedDate' => '2021-10-15',
+                    ],
+                    [
+                        'employee' => [
+                            'empNumber'=> 10,
+                            'lastName' => 'user09',
+                            'firstName'=> 'test',
+                            'deleted'=> false,
+                        ],
+                        'jobTitle' => 'Senior Project Lead',
+                        'joinedDate' => '2020-10-15',
+                    ],
+                ],
+                "meta" => [
+                    "count" => 8,
+                ]
+            ])
+        );
+        $response->headers->set('Content-Type', 'application/json');
+        $response->setStatusCode(Response::HTTP_OK);
+        return $response->send();
+    }
+}
```

Comment:
add new line end of the file. seems missed in self review.
Also use `php-cs-fix` and check